src/System.Private.Interop/src/Shared/WindowsRuntimeMarshal.cs

```diff
@@ -977,8 +977,7 @@ internal class ReaderWriterLockTimedOutException : /*ApplicationException*/ Exce
             {
             }
 
-            /// I borrowed Vance's reader writer lock implementation from his blog as ReaderWriterLockSlim is
-            /// available in System.Core.dll!
```

Comment:
It is not as easy fix in CoreCLR. We should keep your change to update the comment in CoreCLR.